faq.md

```diff
@@ -15,7 +15,7 @@ No other automotive update system has been designed to work in such rigorous sit
 
 
 ### **How does Uptane work with other systems and protocols?**
-Other mechanisms for performing updates, such as XXX, YYY, ZZZ, are compatible with Uptane. Uptane can use any mechanism for data transport and is designed to provide strong security guarantees even if the underlying network or transport mechanism is compromised. If an automaker wants to move to a secure update system, keeping their existing update system as a transport for Uptane is an effective way to do so.
+Other mechanisms for performing updates, such as Red Bend, Movimento, and Tesla, are compatible with Uptane. Uptane can use any mechanism for data transport and is designed to provide strong security guarantees even if the underlying network or transport mechanism is compromised. If an automaker wants to move to a secure update system, keeping their existing update system as a transport for Uptane is an effective way to do so.
```

Comment:
Are these mechanisms or names of companies? Maybe if we said something like "such as those provided by..." that would be clearer to me, but as it is, this sounds a bit clumsy.

Reply:
Corrected. They are companies.